tests/integration_tests/test_kani.py

```diff
@@ -19,15 +21,23 @@ def test_kani(results_dir):
     """
     Test all Kani proof harnesses.
     """
+    # https://psutil.readthedocs.io/en/latest/#psutil.virtual_memory confusingly enough returns
+    # information about the physical memory. Limiting to 64GB should exclude most consumer hardware,
+    # preventing the test from locking up peoples computers, while still allowing them to force their
+    # way through it with enough swap.
+    if (
+        psutil.swap_memory().total + psutil.virtual_memory().total
+        <= 64 * 1000 * 1000 * 1000
+    ):
+        pytest.skip("kani requires at least 64GB RAM")
```

Comment:
Would it make sense to add this like how we skip other tests, e.g.

```python
@pytest.mark.timeout(1800)
@pytest.mark.skipif(PLATFORM != "x86_64", reason="Kani proofs run only on x86_64.")
@pytest.mark.skipif(
    psutil.swap_memory().total + psutil.virtual_memory().total < 64 * 2**30,
    reason="kani requires at least 64GB of memory"
)
def test_kani(results_dir):
```
```

Reply:
The check based on the CI has been moved to use `@pytest.mark.skipif` which covers this point.